target/scripts/startup/setup-stack.sh

```diff
@@ -20,6 +20,16 @@ function _setup() {
     ${FUNC}
   done
 
+  _setup_post
+}
+
+function _setup_post() {
+  # Dovecot `.svbin` files must have a newer mtime than their `.sieve` source files,
+  # Modifications during setup to these files sometimes results in a common mtime value.
+  # Handled during post-setup as setup of Dovecot Sieve scripts is not centralized.
+  find /usr/lib/dovecot/ -iname *.sieve -exec touch -d '2 seconds ago' {} +
+  find /usr/lib/dovecot/ -iname *.svbin -exec touch -d '1 seconds ago' {} +
```

Comment:
Appeasing the linting gods 🙏 

```suggestion
  find /usr/lib/dovecot/ -iname '*.sieve' -exec touch -d '2 seconds ago' {} +
  find /usr/lib/dovecot/ -iname '*.svbin' -exec touch -d '1 seconds ago' {} +
```

Reply:
This is actually very valuable because Bash would expand the star as it is a glob 🌝